go/database/mpt/live_trie_test.go

```diff
@@ -893,6 +893,100 @@ func TestLiveTrie_VerificationOfLiveTrieWithCorruptedFileFails(t *testing.T) {
 	}
 }
 
+func TestLiveTrie_HasEmptyStorage(t *testing.T) {
```

Comment:
this test seems a copy+paste of the test for forest. Since LiveTrie only pass through to the forest, I think it would be enough just to test that a call is routed to the forest, perhaps via a mock.